strategy/json/sql/postgres/single_stream_strategy.go

```diff
@@ -43,32 +45,29 @@ func (s *SingleStreamStrategy) CreateSchema(tableName string) []string {
     event_id UUID NOT NULL,
     event_name VARCHAR(100) NOT NULL,
     payload JSON NOT NULL,
-    metadata JSONB NOT NULL,
+    metadata JSONB NOT NULL, 
+    aggregate_type VARCHAR(50) NOT NULL,
+	aggregate_id UUID NOT NULL,
+	aggregate_version SMALLINT NOT NULL,
     created_at TIMESTAMP(6) NOT NULL,
     PRIMARY KEY (no),
-    CONSTRAINT aggregate_version_not_null CHECK ((metadata->>'_aggregate_version') IS NOT NULL),
-    CONSTRAINT aggregate_type_not_null CHECK ((metadata->>'_aggregate_type') IS NOT NULL),
-    CONSTRAINT aggregate_id_not_null CHECK ((metadata->>'_aggregate_id') IS NOT NULL),
     UNIQUE (event_id)
 );`,
 		tableName,
 	)
-	statements[1] = fmt.Sprintf(
-		`CREATE UNIQUE INDEX ON %s
-((metadata->>'_aggregate_type'), (metadata->>'_aggregate_id'), (metadata->>'_aggregate_version'));`,
-		tableName,
-	)
-	statements[2] = fmt.Sprintf(
-		`CREATE INDEX ON %s
-((metadata->>'_aggregate_type'), (metadata->>'_aggregate_id'), no);`,
-		tableName,
-	)
+	statements[1] = fmt.Sprintf(`CREATE UNIQUE INDEX ON %s (aggregate_type, aggregate_id, no);`, tableName)
```

Comment:
Do we really need `aggregate_type` if we use 1 `event_<aggregate_type>` table per aggregate type? We already know the aggregate_type based on the table we query.

Reply:
Goengine is suppose to support both scenarios (in our case we use different table per aggregate root in which we don't need `aggregate_type` to be part of index), so I guess it's fine to keep this as default.

Reply:
Can we have it perhaps toggle-able so we don't end up with an `aggregate_type` column in the case where we have one table per aggregate type?

Reply:
IMO, making it toggle/configurable is going to introduce some complexity in 
- constructing the migrations, since you can either let goengine do it for you, or add your own. 
-  constructing the queries to retrieve the from the event stream.
- it might also not be that clear if one will/will not have more than one aggregate type at the beginning.

wdyt?

Reply:
Ok, I agree that perhaps the complexity needed for this is too much for this PR. We can think of it another time. :)